src/image/ExposureInfo.cc

```diff
@@ -328,6 +328,7 @@ ExposureInfo::FitsWriteData ExposureInfo::_startWriteFits(lsst::geom::Point2I co
     }
 
     // So long as VisitInfo also writes exposureId (until DM-32138), let ours take precedence.
+    // TODO: what about this one?
```

Comment:
Remove the comments, and possibly move the block higher up? If the `setVisitInfoMetadata` call above no longer sets `EXPID`, then there's no conflict and no reason to care about the order.